torch_ipex/csrc/aten_ipex_bridge.cpp

```diff
@@ -105,6 +104,41 @@ void reorderDilTensorToPublic(const at::Tensor& ipexTensor) {
   }
 }
 
+void reorderDilTensorGeneric(const at::Tensor& ipexTensor, const dil::tensor::desc& dstDesc) {
+  // ipexTensor is not required to be a DIL tensor
+  dil::tensor src = cpu::dbl::comm::try_gen_dil_tensor(ipexTensor);
+  dil::tensor dst {dstDesc};
+  dst.feed_from(src);
+
+  cpu::ShadeDataContext *new_shade_data_context = cpu::ShadeDataContext::allocShadeDataContext();
+  new_shade_data_context->data_type = cpu::SHADE_DATA_TYPE::DIL;
+  new_shade_data_context->dil_tensor = dst;
+
+  if (dstDesc.is_plain()) {
+    // Share with DNNL raw data because it is plain format now
+    new_shade_data_context->cpu_raw_data = dst.get_data_handle();
+    // Cannot free CPU data because the the data is owned by DNNL
+    new_shade_data_context->cpu_del_fun = &(c10::detail::deleteNothing);
+  } else {
+    // If tensor is of blocked format, cpu raw data means nothing here.
+    new_shade_data_context->cpu_raw_data = nullptr;
+    new_shade_data_context->cpu_del_fun = nullptr;
+  }
+
+  // Create a new DataPtr instances because the DataPtr class does not support set
+  // its data or context directly
+  c10::DataPtr shade_data_ptr(
+    dst.get_data_handle(),
```

Comment:
We cannot set the data handle of ideep tensor to Aten tensor data if ideep tensor is with block format. Suggest to use `new_shade_data_context->cpu_raw_data`

Reply:
got it